src/functionalTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerControllerFunctionalTest.java

```diff
@@ -216,34 +216,53 @@ public void should_succeed_and_create_a_task_with_no_categories() {
             "Asylum"
         );
 
-        Response taskFound = findTasksByCaseId(caseId, 1);
-
-        caseId1Task1Id = taskFound
-            .then().assertThat()
-            .body("[0].id", notNullValue())
-            .extract()
-            .path("[0].id");
-
-        taskIdStatusMap.put(caseId1Task1Id, "completed");
+        await().ignoreException(AssertionError.class)
+            .pollInterval(2, SECONDS)
+            .atMost(180, SECONDS)
```

Comment:
Could put these values as constant variables in once place or set them as defaults in awaitility?   That way if you want to change it you can do it one place.